src/planter/planter.controller.ts

```
@@ -3,29 +3,47 @@ import { PlanterService } from './planter.service';
   })
-  @Get(':id')
-  async getUserPlantFromId(@Param('id', ParseIntPipe) id: number, @Request() req) {
+  @Get('/user-plant/:id')
+  async getUserPlant(@Param('id', ParseIntPipe) userPlantId: number, @Request() req) {
     const userId = req.user.userId;
-    return this.planterService.getUserPlantFromPlantId(id, userId);
+    return await this.planterService.getUserPlant(userId, userPlantId);
+  }
+
+  @ApiOperation({
+    summary: "모든 식물 정보 API",
```

ESLint

[error] Replace `"모든·식물·정보·API"` with `'모든·식물·정보·API'`

(prettier/prettier)